lib/setup-env

```diff
@@ -71,20 +71,41 @@ EOF
   fi
 }
 
+find_linux_distribution() {
+  distro=`awk '/^NAME=/' /etc/*-release | sed 's/NAME=//' | tr '[:upper:]' '[:lower:]'`
```

Comment:
`/etc/*-release` exists in all distributions?
On RHEL, `cat /etc/*-release` gives you (for example):
```
Red Hat Enterprise Linux Server release 6.6 (Santiago)
Red Hat Enterprise Linux Server release 6.6 (Santiago)
```
=> No `NAME=`, so `distro` is empty.